domain-mapping/api/class-darkmatter-domains.php

```diff
@@ -171,10 +213,16 @@ public function add( $fqdn = '', $is_primary = false, $is_https = false, $force
 				'%s',
 				'%d',
 				'%d',
+				'%d',
 			)
 		);
 
 		if ( $result ) {
+			/**
+			 * Clear cache but not the one for the newly added domain.
```

Comment:
The following comment seems to be more correct.
```suggestion
			 * Clear internal cache but not for any domains.
```